frontend/src/app/testResults/mocks/queries.mock.tsx

```diff
@@ -33,20 +32,7 @@ import resultForCorrection from "./resultForCorrection";
 export const mocks = [
   {
     request: {
-      query: GetResultsCountByFacilityDocument,
-      variables: {
-        facilityId: "1",
-      },
-    },
-    result: {
-      data: {
-        testResultsCount: testResults.length,
-      },
-    },
-  },
-  {
-    request: {
-      query: GetFacilityResultsMultiplexDocument,
+      query: GetFacilityResultsMultiplexWithCountDocument,
```

Comment:
The diff view is freaking out a bit here because I moved some of the queries around to debug things. None of the mocks are appreciably changing other than the shape / reference to the relevant GQL documents.